e2e/features/requestToken/RequestToken.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Request Token
+  Background: Add an account and navigate to wallet
+    Given I add an account with passphrase "peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready" password "Password1$" name 'test_acc'
+    Then I go to page "wallet"
+
+  Scenario: Request token should generate a copy link
+    Given I click on a button with text "Request"
+    Then I should see "Request tokens"
+    And "lisk_mainchain" should be selected in "Recipient application" dropdown
+    And "Lisk" should be selected in "Token" dropdown
+    And button with text 'Copy link' should be disabled
+    Given I type "10" in "amount"
+    Then button with text "Copy link" should be enabled
+    And I click on a button with text "Copy link"
+    Then Clipboard should contain "lisk://wallet?modal=send&recipient=lskm9syv4wrjcjczpegz65zqxhk2cp9dkejs5wbjb&amount=10&token=0400000000000000&recipientChain=04000000"
+
```

Comment:
Missing scenarios?

`Request token "Copy link" disabled state` when required fields are not provided?

Reply:
exactly. Also consider situations like

- Failure to load supported tokens
- empty amount field
- When a message is provided
- Also assert that the QR code gets blurred when the form are invalid

Reply:
Request token disabled states are in background when opening the modal, this is also the default state when opening the modal @eniolam1000752.

But i have now also added tests for:
- A check that QR image is disabled
- When amount is specifically 0
- When a message is provided

As for this "Failure to load supported tokens". If we are going to add tests for these types of states it would require us to spend unreasonable amount of time/resources for testings. It would require that we mock fail network requests in our scenarios, or am i missing something @eniolam1000752.

Reply:
hmmmm that would be true and it would mean we have to like start setting up mocks which I don't think we should explore now